data/entri.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: entri.app
+description: Entri is a vernacular language learning platform for India, focused on helping people acquire relevant job skills. Entri is part of LearnLaunch, a leading EdTech accelerator based out of Boston, Massachusetts.
+founded: 2015
+based: Service Based
+domain: Edtech
+logo: entri_logo.png
```

Comment:
@coder-oj you made a small mistake. The file name should be `entri_logo.jpg` but here it is `entri_logo.png`